### PR TITLE
Use Akvelon editor (#23415)

### PR DESCRIPTION
**Pull request by alexeyinkin**
- Resolves: #23415 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [x] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/get-started-contributing/#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

File: learning/tour-of-beam/frontend/pubspec.lock [hunk 1]
```diff
@@ -390,6 +402,13 @@ packages:
       url: "https://pub.dartlang.org"
     source: hosted
     version: "0.7.0"
+  hive:
```
Comment (alexeyinkin) [inline learning/tour-of-beam/frontend/pubspec.lock:405]: This was brought by https://pub.dev/packages/autotrie
and will be tree-shaken away.

**Comment by alexeyinkin** [comment]
R: @damondouglas

**Comment by alexeyinkin** [comment]
Fixed the bug that had been moving cursor to the top, https://github.com/akvelon/flutter-code-editor/issues/81

**Review by damondouglas — APPROVED** [review]
@pabloem (cc: @alexeyinkin ) LGTM with the following note:
We discussed in the meeting that the `flutter_code_editor` is soon to be published at https://pub.dev/.

**Comment by alexeyinkin** [comment]
It is public, just unlisted in search. Use the the direct link for it: https://pub.dev/packages/flutter_code_editor
An even the search has "Include unlisted" option.
So the package is pretty much visible.

**Comment by pabloem** [comment]
exciting. thanks!

**Comment by damondouglas** [comment]
Wow I didn't even know that was an option.  No wonder why I was able to build it locally.  🫣

**Pull request merged**